### PR TITLE
MERN_MongoDB: 5.2 Setting up database using Mongoose

### LINKED ISSUE #2684: MERN_MongoDB: 5.2 Setting up database using Mongoose
## Connecting database to the application using mongoose
- **Explain how to connect a local database to nodejs application using mongoose.**

## Project Directory
`Branch MERN>MERN/Topic/Backend/MongoDB/`

### Domain

MERN

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by SMD-1**
<hr>

## Description 📜

This is issue fixes how to connect the database locally using mongoose
Fixes #2684 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [x] MERN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Naman-1234** [comment]
Also please change the name of Pull Request to `MERN_MongoDB: 5.2 Setting up database using Mongoose`.

File: MERN/Topic/Backend/MongoDB/5.2_Connecting_application_to_database_using_Mongoose.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# **Connecting database to the application using mongoose**
+
+MongoDB is the most popular **NoSQL** database in developer's world.  
+If you are working on the NodeJS app, you can connect your Node application to MongoDB using mongoose. **Mongoose** is an Object Data Modelling (ODM) library that basically helps us to establish a connection between MongoDB and Nodejs.
+
+Before using mongoose make sure you have installed:
+
+- Node.js
```
[the remark this answers is not in the document]
Comment (SMD-1) [reply MERN/Topic/Backend/MongoDB/5.2_Connecting_application_to_database_using_Mongoose.md:8]: Done

[the remark this answers is not in the document]
Comment (Naman-1234) [reply MERN/Topic/Backend/MongoDB/5.2_Connecting_application_to_database_using_Mongoose.md:8]: Did you push it as well. I am seeing no further commit after changes requested.

[the remark this answers is not in the document]
Comment (SMD-1) [reply MERN/Topic/Backend/MongoDB/5.2_Connecting_application_to_database_using_Mongoose.md:8]: no i have not, i'll push all together

**Review by saloni-15 — APPROVED** [review]

**Pull request merged**